assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -117,8 +121,16 @@ function useCanvasSelectionCopyPaste(container) {
         if (html) {
           const template = document.createElement('template');
           template.innerHTML = html;
+          let copyingStoryElement = false;
+          let copiedContent = '';
           for (let n = template.content.firstChild; n; n = n.nextSibling) {
             if (n.nodeType !== /* COMMENT */ 8) {
+              if (copiedContent.trim().length && n.tagName === 'P') {
+                copiedContent += ' ';
```

Comment:
Why not `\n`?

Also, what if the content contains `<strong>` and `<i>`, etc? Maybe it's worth trying to pull and cleanup the whole markup?

Reply:
Yep, the intent was to get something super basic to alpha but looks like it's not happening and there' more time -- will rework it!